book/website/reproducible-research/licensing/licensiing-hardware.md

```diff
@@ -0,0 +1,193 @@
+# Open Hardware Licenses- section to be used in the licensing section.
+
+## Introduction
+Licensing is an important aspect of sharing/publishing open source projects as it provides clarity for anyone looking to reuse your open source project.
+Without licenses in place, anyone who wants to reuse it, is left with legal ambiguity as to the status of using your intellectual property.
+While we are looking to focus on open source hardware licenses, this should be considered within the broader open source movement. 
+The open source licensing of hardware came out of open source software licenses and, in fact, several open source software licenses are widely accepted within the open hardware community. 
+
+```{warning}
```

Comment:
This warning feels like something that might be better suited to the general licensing page, as it applies more broadly to all the existing chapters surrounding licenses that we currently have: https://the-turing-way.netlify.app/reproducible-research/licensing.html.

Maybe we could bring this over there, and add a shorter disclosure here?

Reply:
yes, some hypothesis comment asks for it too.